<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="settingsLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="change_rootFolder">
      <property name="text">
       <string>Change...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="executablePathEntry"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="change_executablePath">
      <property name="text">
       <string>Change...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="rootFolderEntry"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_treeViewRootFolder">
      <property name="text">
       <string>Tree view root folder:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_executablePath">
      <property name="text">
       <string>Text editor executable path:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
